ext/hermann/rdcrc32.h

```diff
@@ -0,0 +1,103 @@
+/**
+ * \file rdcrc32.h
+ * Functions and types for CRC checks.
+ *
+ * Generated on Tue May  8 17:36:59 2012,
+ * by pycrc v0.7.10, http://www.tty1.net/pycrc/
```

Comment:
Ah! Good to know, I figure it was something like that. Would you mind adding some commentary in `hermann_rdkafka.c` around this, and noting that this can be removed once the next release of librdkafka is released *and* hermann updates to it?

With that in, I think this looks good for a merge and release